src/core/main.c

```diff
@@ -2124,22 +2124,23 @@ static int do_reexecute(
         args[0] = "/sbin/init";
         (void) execv(args[0], (char* const*) args);
         r = -errno;
-
-        manager_status_printf(NULL, STATUS_TYPE_EMERGENCY,
-                              ANSI_HIGHLIGHT_RED "  !!  " ANSI_NORMAL,
-                              "Failed to execute /sbin/init");
+        *ret_error_message = "Failed to execute /sbin/init";
```

Comment:
move to an else branch below?

Reply:
Nope. It's used in manager_status_prinf().

Reply:
oh, right